server/src/main/scala/com/ossuminc/riddl/lsp/server/RiddlLSPServer.scala

```diff
@@ -1,8 +1,34 @@
 package com.ossuminc.riddl.lsp.server
 
-object RiddlLSPServer {
+import org.eclipse.lsp4j.{CompletionOptions, InitializeParams, InitializeResult, ServerCapabilities, TextDocumentSyncKind}
+import org.eclipse.lsp4j.services.{LanguageServer, TextDocumentService, WorkspaceService}
 
-  final def main(args: Array[String]): Unit = {
-    System.exit(0)
+import java.util.concurrent.CompletableFuture
+
+object RiddlLSPServer extends LanguageServer {
```

Comment:
Please write a scala comment explaining what this thing is and what its base class and various services do.

Reply:
I am thinking it will be best to do that as I build out the server. There are a lot of methods in the API for LanguageServer that could be used. We may want to have a meeting to prioritize what to do. Take a look here:
https://github.com/eclipse-lsp4j/lsp4j/blob/main/org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/services/LanguageServer.java